pkgs/tools/filesystems/garage/default.nix

```diff
@@ -36,24 +40,26 @@ let
       "telemetry-otlp"
       "lmdb"
       "sqlite"
+      "consul-discovery"
     ]);
 
     # To make integration tests pass, we include the optional k2v feature here,
-    # but not in buildFeatures. See:
-    # https://garagehq.deuxfleurs.fr/documentation/reference-manual/k2v/
+    # but in buildFeatures only for version 0.8+, where it's enabled by default.
+    # See: https://garagehq.deuxfleurs.fr/documentation/reference-manual/k2v/
     checkFeatures = [
       "k2v"
       "kubernetes-discovery"
     ] ++
     (lib.optional (lib.versionAtLeast version "0.8") [
       "bundled-libs"
       "sled"
-      "metrics"
-      "telemetry-otlp"
       "lmdb"
       "sqlite"
     ]);
 
+    # Workaround until upstream fixes integration test race condition
```

Comment:
Do they have an issue upstream on this? If so, we should link it there.

Reply:
Just checked again, there's no issue yet. I'll create one and link to it

Reply:
Done